src/pretix/_base_settings.py

```diff
@@ -235,7 +235,12 @@
     )
 }
 
-CURRENCIES = list(currencies)
+CURRENCIES = [
+    c for c in currencies
+    if c.alpha_3 not in {
+        'XAF', 'XAG', 'XAU', 'XBA', 'XBB', 'XBC', 'XBD', 'XDR', 'XOF', 'XPD', 'XPF', 'XPT', 'XSU', 'XTS', 'XUA',
```

Comment:
Why are `XAF`, `XOF`, `XPF` excluded? There are even CURRENCY_PLACES defined for them (see lines 258-260.

Reply:
Good catch, missed those!